step_by_step/README.md

```diff
@@ -2,59 +2,95 @@
-- Make sure attendees confirmed their attendance
-- [Make sure all the logistics are figured out](../logistics/README.md)
+- Announce your coaches on your website, on social media, and in other promotion channels as you wish.
+- [Close the application process and select attendees](../application_form/selection.md). We recommend recruiting a selection panel of at least 3-4 people who can vote and discuss   this, especially in cases where financial aid is involved.
+- [Inform applicants of the result](../application_form/communication.md). You'll want to give the selected applicants a few days to confirm their attendance, and we also recommend that you have a few waiting-list candidates in case of cancellations.
+- Make sure attendees confirmed their attendance. Don't worry if some applicants don't respond by the time that you set, simply move on to the next person on the waiting list.
+- [Verify that all the logistics are figured out](../logistics/README.md).
+- Reach out to potential guest speakers. Often, the workshop attendees come from different walks of life and aren't familiar with many aspects of open-source communities.
+
+### 2 weeks before
+
+- After you receive confirmations from all attendees, assign coaches to their groups and make sure the coaches review the [Coach's manual](http://coach.djangogirls.org/) and the [Tutorial](../tutorial/README.md).
+- Send an email to your coaches so they can meet their group. If you can't do the installation party the evening before the event, invite your coach to do it online with their group following those [instructions](http://tutorial.djangogirls.org/en/installation/index.html). It's really important that attendees' computers are ready for the workshop!
+- Schedule a a coach dinner-meeting the evening before the workshop. This is especially important to be done in advance if you are co-locating the workshop with a conference, since many coaches will need to plan their travel to the workshop in advance.
+- Post the schedule on your Website. You can use the [example schedule](../example_schedule/README.md) template to get an estimate of how much time you'll need for each part of the day.
 
 ### 1 week before
 
-- Confirm with attendees one last time that they are still planning to attend the event! Make some last minute replacements if someone cancels.
-- Send an email to your coaches so they can meet their group. If you can't do the installation party the evening before the event, invite your coach to do it online with their group following those [instructions](http://tutorial.djangogirls.org/en/installation/index.html). It's really important that attendees' computers are ready for the workshop! 
+- Confirm with attendees one last time that they are still planning to attend the event! Make some last minute replacements if someone cancels. At this point you should send the workshop information to your attendees (remember the Web page that you created with locations and schedule?)
```

Comment:
"(remember the Web page that you created with locations and schedule?)" => (schedule, venue details, links to the tutorial, etc).